src/N3DataFactory.js

```diff
@@ -268,6 +269,8 @@ DataFactory = {
     Triple: Quad,
     fromId,
     toId,
+    _resetBlankNodeIds: () => { _blankNodeCounter = 0; },
+    _setBlankNodePrefix: (prefix) => { _blankNodePrefix = prefix; },
```

Comment:
I'm not sure it makes sense to be able to change the blank node prefix. I added this because the tests seemed to expect it.

Reply:
Might still be useful for identification purposes later on (when merging triples from multiple sources).